<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>351</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="ChatStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>430</width>
     <height>331</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>430</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="labelGif">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>430</width>
       <height>180</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>430</width>
       <height>180</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widgetBody" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>430</width>
       <height>150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>430</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 240, 240);</string>
     </property>
     <widget class="QLabel" name="labelUserHead">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>82</width>
        <height>82</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>82</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>82</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QIconLineEdit" name="lineEditUser">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>11</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	border: 1px solid #d6d6d6;
	border-top-right-radius: 5px;
	border-top-left-radius: 5px;
	border-bottom: none;
}</string>
      </property>
      <property name="text">
       <string>xiaoni</string>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QIconLineEdit" name="lineEditPasswd">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>42</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	border: 1px solid #d6d6d6;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>1234</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>用户密码</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPasswd">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>73</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxAutoLogin">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>73</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>100</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
            
	
	background-color: rgb(8, 165, 222);
            border-radius: 4px;
            padding: 8px 16px;
            color: white;
            font-size: 14px;
            border: none;
   }
 QPushButton:hover {
            background-color: rgba(8, 165, 222, 0.6);
        }
QPushButton:pressed {
            background-color: rgba(8, 165, 222, 0.5);
    }

</string>
      </property>
      <property name="text">
       <string>登   录</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnWinMenu">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton
{
	border-image: url(:/resource/images/flicker.png) 0 44 0 0 ;
	background-color: rgba(255, 255, 255, 0);
}

QToolButton:hover
{
	border-image: url(:/resource/images/flicker.png) 0 0 0 44;
background-color: rgba(255, 255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRegister">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	color: #2184e7;	
	min-height: 20px;
}
</string>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnLoginClose">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image: url(:/resource/images/closeback.png) 0 60 0 0 ;
}

QPushButton:hover
{
	border-image: url(:/resource/images/closeback.png) 0 0 0 60;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 240, 240);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="0" columnstretch="0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="widgetBody_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetWinTitle_2" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 162, 191);
color: rgb(255, 255, 255);
border-top-left-radius: 4px; /* 左上角圆角 */
    border-top-right-radius: 4px; /* 右上角圆角 */
border-bottom-left-radius: 0px; /* 左下角圆角 */
    border-bottom-right-radius: 0px; /* 右下角圆角 */</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_WinTitle_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>   登陆配置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnWinMin_2">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
            
	background-color: rgb(255, 162, 191);
            border-radius: 4px;
            padding: 8px 16px;
            color: white;
            font-size: 14px;
            border: none;
   }
 QPushButton:hover {
            background-color: rgba(255, 162, 191, 0.6);
        }
QPushButton:pressed {
            background-color: rgba(255, 162, 191, 0.5);
    }
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/resource/common/ic_min_white.png</normaloff>:/resource/common/ic_min_white.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnWinClose_2">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
            
	background-color: rgb(255, 162, 191);
            border-radius: 4px;
            padding: 8px 16px;
            color: white;
            font-size: 14px;
            border: none;
   }
 QPushButton:hover {
            background-color: rgba(255, 162, 191, 0.6);
        }
QPushButton:pressed {
            background-color: rgba(255, 162, 191, 0.5);
    }
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/resource/common/ic_close_white.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="title">
           <string>服务器设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="verticalSpacing">
              <number>20</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>服务器地址配置：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="IPLineEidt" name="lineEditHostAddr">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius: 0px; /* 左上角圆角 */
    border-top-right-radius: 0px; /* 右上角圆角 */
    border-bottom-left-radius: 0px; /* 左下角圆角 */
    border-bottom-right-radius: 0px; /* 右下角圆角 */</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>消息服务器端口：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_HostMsgPort">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);border-top-left-radius: 0px; /* 左上角圆角 */
    border-top-right-radius: 0px; /* 右上角圆角 */
    border-bottom-left-radius: 0px; /* 左下角圆角 */
    border-bottom-right-radius: 0px; /* 右下角圆角 */</string>
               </property>
               <property name="text">
                <string>60001</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>文件服务器端口：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_HostFilePort">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius: 0px; /* 左上角圆角 */
    border-top-right-radius: 0px; /* 右上角圆角 */
    border-bottom-left-radius: 0px; /* 左下角圆角 */
    border-bottom-right-radius: 0px; /* 右下角圆角 */</string>
               </property>
               <property name="text">
                <string>60002</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>98</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,1,1">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
            
	
	background-color: rgb(8, 165, 222);
            border-radius: 4px;
            padding: 8px 16px;
            color: white;
            font-size: 14px;
            border: none;
   }
 QPushButton:hover {
            background-color: rgba(8, 165, 222, 0.6);
        }
QPushButton:pressed {
            background-color: rgba(8, 165, 222, 0.5);
    }

</string>
            </property>
            <property name="text">
             <string>返回</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveFCfg">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
            
	
	background-color: rgb(8, 165, 222);
            border-radius: 4px;
            padding: 8px 16px;
            color: white;
            font-size: 14px;
            border: none;
   }
 QPushButton:hover {
            background-color: rgba(8, 165, 222, 0.6);
        }
QPushButton:pressed {
            background-color: rgba(8, 165, 222, 0.5);
    }

</string>
            </property>
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">chatstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IPLineEidt</class>
   <extends>QLineEdit</extends>
   <header location="global">iplineeidt.h</header>
  </customwidget>
  <customwidget>
   <class>QIconLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnLoginClose</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWinMin_2</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWinClose_2</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
